printer_mods/ZZP8202/Pull-out_electrical_store/README.md

```diff
@@ -1,42 +1,69 @@
-# Voron 2.4 Pull-out electrical Store 抽拉式电器仓
+# Voron 2.4 Pull-out electrical Store 
 
-## 适用
 
-| Creator | Mod title | Description | V0 | V1.8 | V2.4 | VSW | Trident |
-| --- | --- | --- | --- | --- | --- | --- | --- |
-|ZZP8202|[Pull-out_electrical_store](https://github.com/zzp8202/Pull-out_electrical_store)| Voron 2.4 Pull-out electrical Store |:x:|:x:|:heavy_check_mark:|:x:|:x:|
+## Important Notice
 
-Only aimed at Voron V2.4 350mm printer now.
+**Only aimed at Voron V2.4 350mm printer now.**  
+    
 
-目前只适用于Voron V2.4 350mm 机型。
+* Due to the need to ensure a smooth drawer out, after using this MOD will reduce the space of the electrical store, please check the below table find the available space.
 
+>| **Printer spec**|Inner space     |**Available space**|
+>| ----------------| :------------: | :---------------: |
+>| 350mm           | 470 * 285 * 77 | 470 * 257 * 60    |
+>| 300mm           | 420 * 235 * 77 | 420 * 207 * 60    |
+>| 250mm           | 370 * 185 * 77 | 370 * 157 * 60    |
 
-## Overall 总览
+* You will need about 30 metres of additional wire to connect the drawers.
 
-### Picture
 
-![图片](Photos/Voron_V2.4_Pull-out_electrical_store.PNG)
-### video
-https://www.bilibili.com/video/BV1iS4y1w7My?share_source=copy_web
+Please use it according to the actual situation.
 
-https://www.youtube.com/embed/usf8aM6o3Y0
+## Objectives
 
-https://www.bilibili.com/video/BV12i4y1U7ux?share_source=copy_web
+The objectives for this project are:
 
-https://www.youtube.com/embed/9XUpTigEtQ4
+* Easily to open the electrical bin for debugging
 
-## Objectives
+## Overall
 
-The objectives for this project are:
-此项目的目标是：
+### Picture
+
+![图片](Photos/Voron_V2.4_Pull-out_electrical_store.PNG)
+### Video
+
+https://www.youtube.com/embed/usf8aM6o3Y0
 
-- 装逼* Just cool
-- 
+https://www.youtube.com/embed/9XUpTigEtQ4
+ 
 ## Bill of Materials (BOM)
 
-待补充 to be supplemented
+>| **Category**|Part Name |**QTY**|Remark
+>| ----------------| :------------ | :--------------- | :--------------- |
+>|Printed Parts|Mini12864_Case|X1||
+>|Printed Parts|Front panel L|X1||
+>|Printed Parts|Front panel R|X1||
+>|Cables|24AWG Silicone Cable| X26M |Additional |
+>|Cables|20AWG Silicone Cable| X4M |Additional |
+>|Cables|[Cable Chain](https://www.amazon.com/Befenybay-Internal-Flexible-Machines-10mmX20mm/dp/B07SFFT1K5/ref=sr_1_3?crid=39NLBNYTC6OZX&keywords=Cable%2BChain&qid=1650390801&sprefix=cable%2Bchain%2Caps%2C975&sr=8-3&th=1)|X1|10*20(Inner H x Inner W)  (R18max)|

```

Comment:
everything except `https://www.amazon.com/Befenybay-Internal-Flexible-Machines-10mmX20mm/dp/B07SFFT1K5` can be deleted from this link. it's just tracking data.